src/libraries/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -322,9 +347,10 @@ public void WindowsSimpleWhiteSpaceAsPath_ThrowsArgumentException(string path)
         [Theory,
             MemberData(nameof(ControlWhiteSpace))]
         [PlatformSpecific(TestPlatforms.Windows)]
-        public void WindowsWhiteSpaceAsPath_ThrowsIOException_Core(string path)
+        public void WindowsWhiteSpaceAsPath_ThrowsIOException(string path)
         {
-            Assert.Throws<IOException>(() => Create(path));
+            // Same exception message, different exception type
```

Comment:
What is this referring to?  Is this comment even needed at all?

Reply:
I think it is missing "exception type between Framework and Core". Probably better to break this out.

Reply:
I guess the comment is not necessary when we explicitly call `AssertExtensions.Throws<NetCoreExceptionType, NetFrameworkExceptionType>`. I added the comment to every manual test I made, to track the unit tests where an exception was thrown, and the message string was the exact same, just the exception type differed.